addon/styles/addon.css

```diff
@@ -26,14 +26,14 @@
     padding: 0.2em 1.5em 0em;
     position: absolute;
     white-space: nowrap;
-    top: 2px;
+    top: 0;
     transform-origin: bottom left;
     background-color: #63A9FA;
     color: white;
+    pointer-events: none;
 }
 
 .overlay-border {
-    padding: 10px;
```

Comment:
Does this mean that if people want the existing look they now need to pass `expandSizeBy=10`? If yes, can we make that be the default?

Reply:
@ef4 good call. all set.